pretix_eth/static/pretix_eth/question-type.js

```diff
@@ -1,7 +1,58 @@
+"use strict";
+
 /*
  * payment_eth_html_add_question_type_javascript
  */
 
+// Unpkg imports
+const Web3Modal = window.Web3Modal.default;
+const WalletConnectProvider = window.WalletConnectProvider.default;
+const EvmChains = window.EvmChains;
+
+// Web3modal instance
+let web3Modal;
+
+// Chosen wallet provider given by the dialog window
+let provider;
+
+// Address of the selected account
+let selectedAccount;
+
+function init() {
+  const providerOptions = {
+    walletconnect: {
+      package: WalletConnectProvider,
+    },
+  };
+
+  web3Modal = new Web3Modal({
+    cacheProvider: false, // optional
+    providerOptions, // required
+  });
+}
+
+async function refreshAccountData() {
+  const web3 = new Web3(provider);
+  const accounts = await web3.eth.getAccounts();
+  const targetedInputElement = document.querySelector(`#${get_payment_eth_question_ids('payment_eth_info')}`);
```

Comment:
You will have to iterate through this, they can multiple of them. Either if someone had created the address question multiple times in the admin (probably not intended) or if someone is purchasing more than one ticket (possible).